core/background/main.js

```diff
@@ -80,7 +80,7 @@ require([
 
 			if (!isActiveSession) {
 				isActiveSession = true;
-				GA.send('pageview', '/background-injected?version=' + chrome.app.getDetails().version);
+				GA.send('pageview', '/background-injected?version=' + chrome.runtime.getManifest().version);
```

Comment:
How does Firefox handle this? 

I was reading a bit about: https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/runtime/getManifest

and in their examples they use:

```
browser.runtime.getManifest()
```